package.json

```diff
@@ -20,12 +20,14 @@
     "classnames": "^2.2.6",
     "core-js": "^2.6.11",
     "debug": "^3.2.6",
+    "election-compass-match": "^2.0.0",
     "express": "^4.17.1",
     "history": "^4.10.1",
     "ioredis": "^4.27.9",
     "isomorphic-style-loader": "^3.0.0",
     "kennitala": "^1.2.4",
     "lodash": "^4.17.20",
+    "lzjs": "^1.3.1",
```

Comment:
I don't see this being used in the rest of the diff.

Reply:
great catch